src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -50,37 +50,37 @@
 
     private ScriptContingencyListRepository scriptContingencyListRepository;
 
-    private FiltersContingencyListRepository filtersContingencyListRepository;
+    private FormContingencyListRepository formContingencyListRepository;
 
     private NetworkStoreService networkStoreService;
 
     private final PathMatcher antMatcher = new AntPathMatcher("\0");
 
-    private final FiltersToGroovyScript filtersToScript = new FiltersToGroovyScript();
+    private final FormToGroovyScript formToScript = new FormToGroovyScript();
 
     // Self injection for @transactional support in internal calls to other methods of this service
     @Autowired
     ContingencyListService self;
 
     public ContingencyListService(ScriptContingencyListRepository scriptContingencyListRepository,
-                                  FiltersContingencyListRepository filtersContingencyListRepository,
+                                  FormContingencyListRepository formContingencyListRepository,
                                   NetworkStoreService networkStoreService) {
         this.scriptContingencyListRepository = scriptContingencyListRepository;
-        this.filtersContingencyListRepository = filtersContingencyListRepository;
+        this.formContingencyListRepository = formContingencyListRepository;
         this.networkStoreService = networkStoreService;
     }
 
     private static ScriptContingencyList fromScriptContingencyListEntity(ScriptContingencyListEntity entity) {
         return new ScriptContingencyList(entity.getId(), entity.getScript() != null ? entity.getScript() : "");
     }
 
-    private static FiltersContingencyList fromFilterContingencyListEntity(FiltersContingencyListEntity entity) {
-        return new FiltersContingencyList(entity.getId(), entity.getEquipmentId(), entity.getEquipmentName(),
+    private static FormContingencyList fromFormContingencyListEntity(FormContingencyListEntity entity) {
+        return new FormContingencyList(entity.getId(), entity.getEquipmentId(), entity.getEquipmentName(),
             entity.getEquipmentType(), entity.getNominalVoltage(), entity.getNominalVoltageOperator(), entity.getCountries());
     }
 
-    private static FiltersContingencyListAttributes fromFilterContingencyListEntityAttributes(FiltersContingencyListEntity entity) {
-        return new FiltersContingencyListAttributes(entity.getId(), entity.getEquipmentId(), entity.getEquipmentName(),
+    private static FormContingencyList fromFormContingencyListEntityAttributes(FormContingencyListEntity entity) {
```

Comment:
Methods fromFormContingencyListEntity and fromFormContingencyListEntityAttributes are the same. Remove one of them.

Reply:
Done